contents/newsletter/failure-modes-of-a-team-lead.md

```diff
@@ -0,0 +1,142 @@
+
+> 🏆 **You'll know you're successful if:** Your team geunienly look up to you. Respect is earned in pull requests far faster than in meetings.
+
+## 3. You're scared of changing goals
+
+### Symptom
+
+Mid-quarter you discover customers aren't using Feature X, but you ship the next phase anyway because it's on the roadmap. By the time the quarter ends, the team has delivered everything except impact.
+
+### Why it happens
+
+Plans feel like commitments. You fear looking indecisive or a failure if you change them. You forget the job of the team lead is to ensure the team is delivering impact, not just code.
+
+### How to fix it
+
+Add a regular feedback loop to evaluate your progress. At PostHog, we do [monthly growth reviews](https://posthog.com/newsletter/product-management-is-broken#b-product-managers-run-monthly-growth-reviews):
+
+First, we gather all available data, such as revenue metrics, product usage, and user feedback from interviews and surveys. Then, the team meets once a month to discuss hard questions like:
+
+- Are our 10 biggest customers happy users of the product?
+- Where is churn spiking and why?
+- Are new releases shifting core metrics or just vanity stats?
+- What surprised us in user interviews?
+- Where are users struggling?
+
+This paints a full picture of how the product and team are doing. It's then up to you to decide if the team shound continue on their course or if something needs to change.
+
+> 🏆 **You'll know you're successful if:** Your reviews end with projects being removed from the roadmap because the data showed better opportunities. At the end of the quarter, your north-star metric is higher than ever.
+
+## 4. You don't keep the talent bar high 
+
+### Symptom
+
+The same bugs and sloppy PRs keep reappearing from the same people, yet no one seems surprised.
+
+### Why it happens
+
+You're scared of giving [hard feedback](/newsletter/how-to-give-feedback), so mediocrity coasts by unchecked. Gradually, your expectations blur and buggy code feels "good enough". Before you know it, you're firefighting the same bugs every sprint instead of shipping new value.
+
+### How to fix it
+
+Great teams stay great by keeping the bar high. Run the Keeper Test quarterly: "If this person resigned today, would I fight to keep them". Dig in where the answer is "no" and ask yourself what would it take for this to be a "yes"? 
+
+Is it just temporary, or is there a deeper issue to resolve? If not, do something today: coaching, clearer expectations, or, when necessary, parting ways. 
+
+Side note: anyone can ask their manager or tech lead "how hard would you work to change my mind if I were thinking of leaving?"". It's a great way to solicit valuable feedback.
+
+> 🏆 **You'll know you're successful if:** Every Keeper Test answer is a confident "yes" and repeat-bug tickets have vanished from the board.
+
+## 5. You're always the hero
```

Comment:
With this framing, this one feels a lot like "being the bottleneck". I think this can mostly stay the same, maybe just adjust the framing to be around burning out. 

- You have a lot of pressure to deliver and engineers relying on you, you take on too much
- This happens because you don't prioritize well, don't ask for help when you need it, don't understand your constraints.
- Fix it by communicating with those above and below you about how you are prioritizing, improving on understanding constraints.